src/python/pants/backend/python/macros/deprecation_fixers.py

```diff
@@ -102,12 +105,17 @@ async def determine_macro_changes(all_targets: AllTargets, _: MacroRenamesReques
         python_requirement_dependencies_fields, build_file_addresses_per_tgt, deps_per_tgt
     ):
         generator_tgt = next(
-            (tgt for tgt in deps if isinstance(tgt, PythonRequirementsFileTarget)), None
+            (tgt for tgt in deps if isinstance(tgt, TargetGeneratorSourcesHelperTarget)), None
         )
         if generator_tgt is None:
             continue
 
-        generator_source = generator_tgt[SingleSourceField].value
+        generator_sources = generator_tgt[TargetGeneratorSourcesHelperSourcesField].value
```

Comment:
I had to change to `sources: list[str]` because Go will need `['go.mod', 'go.sum']`